app/blog/2020-09-22-lit-element-and-lit-html-next-preview.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "LitElement 3.0 & lit-html 2.0: Early Preview Release"
+---
+
+Today we're publishing the first preview releases of the next major versions of our flagship libraries, LitElement and lit-html.
+
+These releases include most of the breaking changes we intend to make, and most of the functionality we want to carry over from the previous versions. They are not yet feature complete or fully API stable. Notably, they don't yet support legacy browsers like IE11, or browsers requiring the web components polyfill.
+
+## Motivation
+
+We've been very happy with the current version of our libraries - they're fast, small, and stable (yay!) - and in some ways we _don't_ have very many pressing needs to make breaking changes. We don't take making breaking changes lightly. But there are some compelling reasons for changes that we think will improve the user-experience of components and applications built with LitElement.
```

Comment:
```suggestion
We've been very happy with the current version of our libraries—they're fast, small, and stable (yay!)—and in some ways we _don't_ have very many pressing needs to make breaking changes. We don't make breaking changes lightly. But there are some compelling reasons for changes that we think will improve the user-experience of components and applications built with LitElement.
```
"take making breaking" has a funny kind of ring to it. Suggest shortening this and replacing hyphens with em-dashes to match our general usage.

Reply:
done